### PR TITLE
add scheduled-backup test case

### PR DESCRIPTION
**Pull request by shuijing198799**
add scheduler test case 
1. add scheduler test case
2.add a function to get backup dir
@weekface @zyguan @xiaojingchen @tennix @cwen0 PTAL

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -252,6 +260,11 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterInfo) error {
 		}
 	}
 
+	_, err := oa.kubeCli.CoreV1().Pods(info.Namespace).Get(getBackupDirPodName, metav1.GetOptions{})
```
Comment (weekface) [inline tests/actions.go]: helm del won't delete these pods?

File: tests/actions.go [hunk 2]
```diff
@@ -1065,13 +1087,196 @@ func releaseIsExist(err error) bool {
...
+
+		return true, nil
+	}
+
+	err := wait.Poll(DefaultPollInterval, DefaultPollTimeout, fn)
+	if err != nil {
+		return fmt.Errorf("failed to launch scheduler backup job: %v", err)
+	}
+
+	backupDir, err = oa.getBackupDir(info)
```
Comment (weekface) [inline tests/actions.go]: Don't use global var `backupDir`.

File: tests/actions.go [hunk 3]
```diff
@@ -1065,13 +1087,196 @@ func releaseIsExist(err error) bool {
...
+	glog.Infof(cmd)
+	res, err := exec.Command("/bin/sh", "-c", cmd).CombinedOutput()
+	if err != nil {
+		glog.Errorf("cluster:[%s/%s] exec :%s failed,error:%v,result:%s", info.Namespace, info.ClusterName, cmd, err, res)
+		return "", err
+	}
+
+	dirs := strings.Split(string(res), "\n")
+	glog.Infof("dirs in pod info name [%s] dir name [%s]", info.Name, strings.Join(dirs, ","))
+	return strings.TrimSpace(dirs[0]), nil
```
Comment (weekface) [inline tests/actions.go]: should assert the count of dirs

File: tests/backup/backupcase.go [hunk 4]
```diff
@@ -70,5 +70,20 @@ func (bc *BackupCase) Run() error {
 		return err
 	}
 
+	bc.srcCluster.Name = "demo-scheduled-backup"
+	bc.desCluster.Name = "demo-scheduled-backup"
```
Comment (weekface) [inline tests/backup/backupcase.go]: don't need desCluster

File: tests/cmd/e2e/main.go [hunk 5]
```diff
@@ -88,6 +93,7 @@ func main() {
 	}
 
 	restoreClusterInfo := &tests.TidbClusterInfo{
+		Name:             "test-backup",
```
Comment (xiaojingchen) [inline tests/cmd/e2e/main.go]: why the name as same as `clusterInfo` name?

File: tests/actions.go (hunk 1 above)
Comment (shuijing198799) [reply tests/actions.go]: this pod is create for get dirs, not manager by helm

File: tests/actions.go (hunk 2 above)
Comment (shuijing198799) [reply tests/actions.go]: done!

File: tests/actions.go (hunk 3 above)
Comment (shuijing198799) [reply tests/actions.go]: done!

File: tests/backup/backupcase.go (hunk 4 above)
Comment (shuijing198799) [reply tests/backup/backupcase.go]: done!

File: tests/cmd/e2e/main.go (hunk 5 above)
Comment (shuijing198799) [reply tests/cmd/e2e/main.go]: the name is pvc name, clusterinfo name is clustername in the test case

File: tests/actions.go [hunk 6]
```diff
@@ -49,14 +49,10 @@ func NewOperatorActions(cli versioned.Interface, kubeCli kubernetes.Interface) O
 
 const (
 	DefaultPollTimeout  time.Duration = 10 * time.Minute
-	DefaultPollInterval time.Duration = 1 * time.Minute
+	DefaultPollInterval time.Duration = 10 * time.Second
```
Comment (weekface) [inline tests/actions.go]: Why change this to 10 seconds, it will be very noisy

File: tests/actions.go [hunk 7]
```diff
@@ -918,11 +913,6 @@ func (oa *operatorActions) CheckAdHocBackup(info *TidbClusterInfo) error {
 		return fmt.Errorf("failed to launch scheduler backup job: %v", err)
 	}
 
-	backupDir, err = oa.getBackupDir(info)
```
Comment (weekface) [inline tests/actions.go]: you must ensure the returned `backupDir ` is not empty.

File: tests/actions.go [hunk 8]
```diff
@@ -1193,7 +1190,7 @@ func getParentUIDFromJob(j batchv1.Job) (types.UID, bool) {
 	return controllerRef.UID, true
 }
 
-func (oa *operatorActions) getBackupDir(info *TidbClusterInfo) (string, error) {
+func (oa *operatorActions) getBackupDir(info *TidbClusterInfo) (int, error) {
```
Comment (weekface) [inline tests/actions.go]: return a `[]string`?

File: tests/actions.go (hunk 6 above)
Comment (shuijing198799) [reply tests/actions.go]: xiaojing and I need to check the log and scheduledbackup-cronjob is called job every minute

**Pull request merged**